pandas/io/gbq.py

```diff
@@ -152,6 +176,8 @@ def read_gbq(
     # START: new kwargs.  Don't populate unless explicitly set.
     if use_bqstorage_api is not None:
         kwargs["use_bqstorage_api"] = use_bqstorage_api
+
+    kwargs["progress_bar_type"] = progress_bar_type
```

Comment:
To avoid breaking people with `pandas-gbq < 0.12.0`, only populate the `progress_bar_type` kwargs if explicitly set.

```suggestion
    if progress_bar_type is not None:
        kwargs["progress_bar_type"] = progress_bar_type
```

Reply:
thanks @tswast !!